### PR TITLE
Memory game board

### PR DESCRIPTION
**Pull request by tmansuroglu**
#22 I have refactored final code to make it more readable. This is a part of it. So, majority of the code is already reviewed. Also, we agreed with Ammar to not add tests. Something is wrong and Ammar will fix it when all code is merged. Tests will be added during bug bash week.

### CODE REVIEW

File: src/components/memorygame/MemoryGameBoard/index.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      <Row align="middle" className="howToPlayPopoverAndScrollToFactsRow">
+        <Col span={5}>
+          <ScrollToFacts />
+        </Col>
+        <Col span={2} push={17}>
+          <HowToPlayPopover />
+        </Col>
+      </Row>
+      <Row>
+        <GameCards
```
Comment (louisrli) [inline src/components/memorygame/MemoryGameBoard/index.jsx]: unless you completely rewrote the code, notice how the diff on the left side is empty and its saying you added 128 lines.

you want to use git mv to move MemoryGameCards/idnex.js to MemoryGameBoard/index.js, then modify it.

imagine you are me. i am wondering whether the code was completely rewritten or whether you rewrote just parts of it. but now i have to rereview the whole thing.

https://stackoverflow.com/questions/433111/how-to-make-git-mark-a-deleted-and-a-new-file-as-a-file-move

basically you shouldnt be deleting and then copy-pasting to a new file, you should be doing `git mv` followed by `git commit` and that should make the diff show up correctly (unless github is strange about it). but give it a try.

Comment (louisrli) [reply src/components/memorygame/MemoryGameBoard/index.jsx]: what i should see is a rename + a diff of how the code was changed. otherwise i'm rereviewing the whole code.

**Comment by tmansuroglu** [comment]
I closed this by mistake while deleting extra commits

**Comment by majdajroudi** [comment]
is this a PR that we still have to review? Files changed tab is empty

**Comment by tmansuroglu** [comment]
No, I messed this up. There is a open version of this.
